exchange/exchange.go

```diff
@@ -79,6 +81,24 @@ type bidResponseWrapper struct {
 	bidder       openrtb_ext.BidderName
 }
 
+type UUIDGenerator interface {
+	NewUUID() string
```

Comment:
Nitpick: This can just be `New`. The fact that's it a UUID is an implementation detail.

Reply:
Renamed